pkg/domain/infra/abi/play.go

```diff
@@ -1255,10 +1254,12 @@ func (ic *ContainerEngine) PlayKubeDown(ctx context.Context, body io.Reader, opt
 			if deploymentYAML.Spec.Replicas != nil {
 				numReplicas = *deploymentYAML.Spec.Replicas
 			}
-			for i := 0; i < int(numReplicas); i++ {
-				podName := fmt.Sprintf("%s-pod-%d", deploymentName, i)
-				podNames = append(podNames, podName)
+			if numReplicas > 1 {
```

Comment:
This code is in `PlayKubeDown` so I'm not sure the warning is needed. Having said that, if we want to print something, we should rephrase to indicate `down` instead of `creating...`